units/systemd-timesyncd.service.in

```diff
@@ -22,6 +22,7 @@ Wants=time-set.target
 AmbientCapabilities=CAP_SYS_TIME
 BusName=org.freedesktop.timesync1
 CapabilityBoundingSet=CAP_SYS_TIME
+Environment=SYSTEMD_NSS_RESOLVE_VALIDATE=0
```

Comment:
Would it be worth to have a small comment here explaining why it's being disabled, given (technically) it's lowering security? Users on the system will only see the unit, not our commit history